setup.py

```diff
@@ -24,9 +24,9 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        'Flask==0.10.1',
```

Comment:
Looks like we can just remove flask from the deps here

Nothing in this package uses it and it was added to the requirements file in 2015

Reply:
Ignore, it's in tests. SMH

Reply:
If it's only in tests we can just move it to `requirements-dev` and leave it out of `setup.py`

Reply:
🙌